src/asset_folder_importer/asset_folder_verify_files/update_vs_thread.py

```diff
@@ -110,7 +110,8 @@ def run(self):
         :return:
         """
         while True:
-            (prio, item) = self._q.get(timeout=self.timeout)
+            queue_item = self._q.get(timeout=self.timeout)
+            item = queue_item.item
```

Comment:
this line needs to go AFTER the if below.  It will cause an exception if queue_item is None

Reply:
The if statement below requires the item object to exist to work so your comment makes no sense.

Reply:
ah, right, i misread it. if `queue_item` is _always_ non-null then you can get rid of it completely:
```
item = self._q.get(timeout=self.timeout).item
```

Reply:
Done.